src/our-work.html

```diff
@@ -109,7 +109,7 @@ <h3 class="card-title fw-boldest pb-4 mb-0 pb-lg-2 mb-lg-5 pt-2 pt-lg-0 mt-4">{{
                 <p class="card-text pb-2 mb-5">{{ featured_work.description }}</p>
               </div>
 
-              <div class="outcome">
+              <div>
```

Comment:
Removed this class because it was purely informational for me to see what's in the Outcome section and above it. Had no styles.